moveit_planners/test_configs/prbt_ikfast_manipulator_plugin/src/prbt_manipulator_ikfast_moveit_plugin.cpp

```diff
@@ -1049,9 +1052,10 @@ bool IKFastKinematicsPlugin::searchPositionIK(const geometry_msgs::msg::Pose& ik
                 best_solution = solution;
               }
             }
-            else
+            else {
               // Return first feasible solution
               return true;
+}
```

Comment:
If you run pre-commit does the formatting of this get fixed?

Reply:
Unfortunately no.  But I just fixed this manually.